src/cli/commands/audit.js

```diff
@@ -115,14 +119,19 @@ export type AuditActionRecommendation = {
 export function setFlags(commander: Object) {
   commander.description('Checks for known security issues with the installed packages.');
   commander.option('--summary', 'Only print the summary.');
+  commander.option(
+    '--level <severity>',
+    'Only print advisories with vulnerability severity greater than or equal to provided level',
```

Comment:
Would be nice to put the acceptable values `'info', 'low', 'moderate', 'high', 'critical'` in this description